CONTRIBUTING.md

```diff
@@ -1,11 +1,12 @@
 # Contributing
 
-## Updating bsp4j `*.xtend` files
-
-Whenever `*.xtend` files are updated you need to manually re-generate Java
-source files with the `sbt bsp4j/xtend` task.  For example, while iterating on
-changes in bsp4j you can use
+## Updating protocol
```

Comment:
When reading the contributing guide, I have read the section name 
> Updating bsp4j `*.xtend` files

and immediate thought "nice, I am not touching those so I can skip it".

Please, comment if you know some better way to call this section